src/metadata/copy_var/config.vsh.yaml

```diff
@@ -0,0 +1,68 @@
+name: copy_var
+namespace: "metadata"
+description: |
+  Make a copy of an existing .var field or index to another .var field.
+authors:
+  - __merge__: /src/authors/dorien_roosen.yaml
+    roles: [ author ]
+argument_groups:
+  - name: "MuData Input"
+    arguments:
+      - name: "--input"
+        alternatives: ["-i"]
+        type: file
+        description: Input h5mu file
+        direction: input
+        required: true
+        example: input.h5mu
+      - name: "--modality"
+        type: string
+        default: "rna"
+        required: false
+      - name: "--input_var_key"
```

Comment:
I'm tempted to ask to make this multiple? But we can always adjust later when we notice that multiple calls are required.